sig-node/CONTRIBUTING.md

```diff
@@ -2,24 +2,98 @@
 
 Welcome!
 
-## For Kubernetes Contributions
+Thank you for your interest in contributing to SIG Node. SIG Node is one of the biggest SIGs in Kubernetes.
+Reliability and performance of millions of nodes running critical applications in production rely on the quality of your contribution(s).
+The diversity of workloads, environments, and the scale SIG Node needs to support makes every code change risky as all the side effects need to be evaluated.
+And the contribution can realistically only be evaluated by a small set of maintainers.
```

Comment:
I'm not a huge fan of this clause and sentence

```
as all the side effects need to be evaluated.
And the contribution can realistically only be evaluated by a small set of maintainers.
```

I think the broader point you're trying to make is the bar is very high for a contributor and maintainers have low bandwidth to fully work through all the possible reprocussions of a contribution. I think that's true and it can be spelled out, but I think it's tricky to word it without fully discouraging contributions you know?

Reply:
plus the first clause 
```
The diversity of workloads, environments, and the scale SIG Node needs to support makes every code change risky
```
 captures that intent